doc/openms/docs/getting-started/knime-get-started.md

```
@@ -0,0 +1,30 @@
+["openms"](https://hub.knime.com/search?q=openms).
+
+
+## Installing KNIME and creating workflows
+
+Read through a introduction to OpenMS in KNIME [user tutorial](/tutorials/knime-user-tutorial.md) containing hands-on training material covering also basic
+usage of KNIME and how to install it. See the official [KNIME Getting Started Guide](https://www.knime.com/getting-started-guide) for a more in-depth view of
+the KNIME functionality besides OpenMS.
+
+If you face any issues, please [contact us](/about/communication.md) and specifically for the usage of OpenMS in KNIME, the KNIME community contribution [forum](https://forum.knime.com/tag/openms).
+
+Click [here](knime/minimal-workflow.md) to create your own minimal workflow.
```

markdownlint-cli2 (0.22.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)